src/panels/config/lovelace/ha-config-lovelace.ts

```diff
@@ -8,11 +8,15 @@ import { HomeAssistant } from "../../../types";
 
 export const lovelaceTabs = [
   {
-    component: "lovelace",
     path: "/config/lovelace/dashboards",
     translationKey: "ui.panel.config.lovelace.dashboards.caption",
     iconPath: mdiViewDashboard,
   },
+  {
+    path: "/config/lovelace/resources",
+    translationKey: "ui.panel.config.lovelace.resources.caption",
+    iconPath: mdiViewDashboard,
+  },
```

Comment:
This will cause `resources` to become a tab on both lovelace config pages, which is something that is not desired.

These config pages should not be using the `tabs-subpage` layout, but just the `subpage` as they don't have tabs.